### ISSUE TITLE
Community Blog background tiling doesn't work on UIToolkit glazier

### ISSUE THREAD

**Issue by DiFFoZ** [SmartlyDressedGames/Unturned-3.x-Community#4800]

**Comment by SDGNelson** [comment]
Hey DiFFoZ, thanks for the report! Unfortunately, at least when I last looked into it, there wasn't support for tiling images in UIToolkit. We needed it for the in-game inventory display so I hacked in a `TileRepeatHintForUITK` property that specified the number of grid cells to tile those images. The main menu doesn't know how many times to tile, but I didn't worry about it because I didn't think anyone was using the UIToolkit implementation at the moment. Do you actively use the UIToolkit UI? :O

**Comment by DiFFoZ** [comment]
> Do you actively use the UIToolkit UI? :O

Yes, after changes in how news feed is rendered, UGUI is causing really bad lag spikes compared to UIToolKit.

**Comment by SDGNelson** [comment]
Fair - I wish uGUI had something akin to Unreal's "Invalidation Box" to cache the news feed contents.
To somewhat mitigate it perhaps I should change it to only load one news post with a button at the bottom to load more? :/

Ideally we'd make UIToolkit the default considering it does perform better, but IIRC there were some differences I couldn't work around in the current version, like the extra space created by labels on the in-game map pushing out the scroll viewport in an undesired way.

I'm still glad to have done the UIToolkit integration since it made me comfortable to start abstracting the automatic layout stuff.

**Comment by SDGNelson** [comment]
Sorry, forgot to do this sooner!
